src/prognostic_equations/eddy_diffusion_closures.jl

```diff
@@ -527,6 +530,90 @@ function mixing_length(
     return MixingLength{FT}(l_final, l_W, l_TKE, l_N, l_grid)
 end
 
+function mixing_length(
+    params,
+    ustar,
+    ᶜz,
+    z_sfc,
+    ᶜdz,
+    sfc_tke,
+    ᶜN²_eff,
+    ᶜtke,
+    obukhov_length,
+    ᶜstrain_rate_norm,
+    ᶜPr,
+    ᶜtke_exch,
+    scale_blending_method,
+    property::Val{P},
+) where {P}
+
+    ᶜmixing_length_tuple = mixing_length_lopez_gomez_2020(
+        params,
+        ustar,
+        ᶜz,
+        z_sfc,
+        ᶜdz,
+        sfc_tke,
+        ᶜN²_eff,
+        ᶜtke,
+        obukhov_length,
+        ᶜstrain_rate_norm,
+        ᶜPr,
+        ᶜtke_exch,
+        scale_blending_method,
+    )
+
+    return getproperty(ᶜmixing_length_tuple, P)
+end
+
+function mixing_length(Y, p, mixing_length_property = :master)
```

Comment:
This is going to be type-unstable if the compiler doesn't feel like constant propagating the `mixing_length_property` symbol. If you want this to be type-stable, I suggest just returning the master mixing length from this function, and defining a separate function `mixing_length_components` that returns a `NamedTuple` of the `wall`, `tke`, and `buoy` components.